base/timing.jl

```diff
@@ -114,6 +114,25 @@ function jit_total_bytes()
     return ccall(:jl_jit_total_bytes, Csize_t, ())
 end
 
+thread_up_time() = ccall(:jl_thread_up_time, UInt64, ())
+thread_user_time() = ccall(:jl_thread_user_time, UInt64, ())
+# thread_user_time(tid::Integer) = ccall(:jl_thread_user_time, UInt64, (Cint,), Cint(tid))
+# function thread_user_time(pool::Symbol=:all)
```

Comment:
i'm thinking an interface like this, where you can optionally return stats by threadpool makes sense?

we also need to change this to be able to return specific stats (like `sleep_time`, `gc_time` etc.) rather than just `user_time`

and planning to do the aggregation on the Julia side, as you can see